README.md

```diff
@@ -305,6 +305,14 @@ dapr upgrade -k --runtime-version=1.0.0 --set global.tag=my-tag --set dapr_opera
 
 *Note: do not use the `dapr upgrade` command if you're upgrading from 0.x versions of Dapr*
 
+### Use Private Helm Repository
+
+export HELM_CHART_REPO_URL="https://helmchart-repo.xxx.xxx/dapr/dapr"
+export HELM_CHART_REPO_USERNAME="username_xxx"
```

Comment:
If this is only specific to DAPR, I am thinking we should prefix the env var name by DAPR to maintain consistency across all other env var. So it should be like `DAPR_HELM_REPO_URL`, `DAPR_HELM_REPO_USERNAME` and `DAPR_HELM_REPO_PASSWORD`
@pravinpushkar wdyt? 
/cc @1046102779 @yaron2

Reply:
I agree with this. `HELM_CHART_REPO_URL` does have the complete url for dapr chart so it makes sense to prefix it with `DAPR`.

Reply:
OK. I instead start with the `DAPR` prefix.